packages/forgae-lib/network.ts

```diff
@@ -0,0 +1,23 @@
+import * as utils from 'forgae-utils'
+
+type INetworkParams = {
```

Comment:
remove this whole section because this get network should be in the utils. Later we will rework the utils on ts